Application/Setting/Univention/Frontend.php

```diff
@@ -0,0 +1,438 @@
+<?php
+namespace SPHERE\Application\Setting\Univention;
+
+use SPHERE\Application\Education\Lesson\Division\Division;
+use SPHERE\Application\Education\Lesson\Term\Term;
+use SPHERE\Application\Education\School\Type\Type;
+use SPHERE\Application\People\Group\Group;
+use SPHERE\Application\Platform\Gatekeeper\Authorization\Consumer\Consumer;
+use SPHERE\Application\Setting\Authorization\Account\Account;
+use SPHERE\Common\Frontend\Icon\Repository\ChevronLeft;
+use SPHERE\Common\Frontend\Icon\Repository\Download;
+use SPHERE\Common\Frontend\Icon\Repository\Exclamation;
+use SPHERE\Common\Frontend\Icon\Repository\Info;
+use SPHERE\Common\Frontend\Icon\Repository\Minus;
+use SPHERE\Common\Frontend\Icon\Repository\Success as SuccessIcon;
+use SPHERE\Common\Frontend\Icon\Repository\Upload;
+use SPHERE\Common\Frontend\IFrontendInterface;
+use SPHERE\Common\Frontend\Layout\Repository\Panel;
+use SPHERE\Common\Frontend\Layout\Repository\Title;
+use SPHERE\Common\Frontend\Layout\Structure\Layout;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutColumn;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutGroup;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutRow;
+use SPHERE\Common\Frontend\Link\Repository\Standard;
+use SPHERE\Common\Frontend\Text\Repository\Bold;
+use SPHERE\Common\Frontend\Text\Repository\Danger as DangerText;
+use SPHERE\Common\Frontend\Text\Repository\Success as SuccessText;
+use SPHERE\Common\Frontend\Text\Repository\ToolTip;
+use SPHERE\Common\Window\Stage;
+use SPHERE\System\Extension\Extension;
+use SPHERE\System\Extension\Repository\Debugger;
+
+/**
+ * Class Frontend
+ *
+ * @package SPHERE\Application\Setting\Univention
+ */
+class Frontend extends Extension implements IFrontendInterface
+{
+
+    /**
+     * @return Stage
+     */
+    public function frontendUnivention()
+    {
+        $Stage = new Stage('Univention', '');
+        $Stage->addButton(new Standard('Zurück', '/Setting', new ChevronLeft()));
+
+        //ToDO Erklärung der Schnittstelle? + Vorraussetzungen
+
+        return $Stage;
+    }
+
+    /**
+     * @param bool $Upload
+     *
+     * @return Stage
+     */
+    public function frontendUnivAPI($Upload = '')
```

Comment:
Codacy found an issue: [The method frontendUnivAPI() has an NPath complexity of 15912. The configured NPath complexity threshold is 200.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)